api/oneclick.py

```diff
@@ -1027,12 +1497,28 @@ def process_access_document(self, access_document):
         expires = datetime.datetime.utcnow() + datetime.timedelta(minutes=14)
         return content_type, content_link, expires
 
+class MockOneClickAPI(OneClickAPI):
 
-class MockOneClickAPI(BaseMockOneClickAPI, OneClickAPI):
+    @classmethod
+    def make_collection(self, _db):
```

Comment:
I don't think this needs to be its own method -- you can move this into the beginning of `mock_collection` and get rid of the `collection = self.make_collection(_db)` line.